lib/constructs/interface.js

```diff
@@ -348,7 +346,7 @@ class Interface {
         this.stringifier = member;
       }
     }
-    for (const member of this.inheritedMembers(seen)) {
+    for (const member of this.inheritedMembers()) {
```

Comment:
`inheritedMembers` still takes a `seen` parameter (for good reasons, I assume).

Reply:
Well, it does create its own `Set`.

The main reason it takes one is so that **WebIDL2JS** could detect if an interface was present in both the prototype chain and as a legacy mixin.

Since `this.members` no longer takes a `seen` argument, there isn’t any more reason to create the `seen` set ourselves.

Reply:
Aha, I missed that it had a default parameter. Looks good then.